delta/extensions/sources/snap_process_sentinel1.sh

```diff
@@ -22,9 +22,21 @@ export PATH=~/snap/bin:$PATH
 gptPath="gpt"
 
 # Get input parameters
-graphXmlPath="$1"
```

Comment:
Can we move this and the sml file to scripts/convert? The delta folder should only be for python code

Reply:
Never mind, now I see why you did this! I would suggest now to leave the xml file, but get rid of the .sh and just have a system call in python (it is only one command). With my upcoming save_tiffs script having a standalone script should be less important. https://github.com/nasa/delta/pull/90/files#diff-43f7c73743790543ad7fbfeecbb6adc21d5ed70788a85e2203ea22086b381a9e